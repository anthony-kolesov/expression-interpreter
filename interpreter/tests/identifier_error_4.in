# Reference to unknown variable in assignment.
var me = undefined2
