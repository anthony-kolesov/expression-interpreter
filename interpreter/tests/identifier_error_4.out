ERROR:2:Unknown identifier: undefined2
